blockchain/serviceMetadata.go

```diff
@@ -58,7 +58,14 @@ func ServiceMetaData() *ServiceMetadata {
 	} else {
 		//TO DO, have a snetd command to create a default metadata json file, for now just read from a local file
 		// when block chain reading is disabled
-		metadata, err = readServiceMetaDataFromLocalFile("service_metadata.json")
+		if metadata, err = readServiceMetaDataFromLocalFile("service_metadata.json"); err != nil {
+			// If no local file is available be default , then return a dummy metadata to support functionality when blockchain is disabled.
+			log.WithError(err)
+			metadata = &ServiceMetadata{}
```

Comment:
Lets make the method name more descriptive

Reply:
Done